docs/content/en/tasks/monitor/_index.md

```diff
@@ -5,168 +5,320 @@ weight: 6
-
-```bash
-helm upgrade k8ssandra k8ssandra/k8ssandra --set prometheus.ingress.enabled=true,prometheus.ingress.host=localhost
-helm upgrade k8ssandra k8ssandra/k8ssandra --set grafana.ingress.enabled=true,grafana.ingress.host=localhost
+```yaml
+apiVersion: k8ssandra.io/v1alpha1
+kind: K8ssandraCluster
+metadata:
+  name: test
+  namespace: k8ssandra-operator
+spec:
+  cassandra:
+    serverVersion: "4.0.3"
+    serverImage: k8ssandra/cass-management-api:4.0.3
+    storageConfig:
+      cassandraDataVolumeClaimSpec:
+        storageClassName: standard
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 1Gi
+    config:
+      jvmOptions:
+        heapSize: 512M
+    networking: 
+      hostNetwork: false
+    datacenters:
+      - metadata:
+          name: dc1
+        size: 3
+    mgmtAPIHeap: 64Mi 
+  stargate:
+    size: 1
+  reaper:
+    keyspace: reaper_db
 ```
+*Download this manifest [here](k8ssandra.yaml).*
 
-To check the installed or upgraded pods' ready status, without having to submit multiple `kubectl get pods` commands, use `kubectl rollout status`. The command waits up to ten minutes (timeout is configurable) and gives a line of output as one or more pods in the set become ready. The format to check a statefulset (sts) is: 
-
-`kubectl rollout status statefulset ${CLUSTERNAME}-${DATACENTER}-default-sts`
+Wait for the pods to come up in the `k8ssandra-operator` namespace and fully start.
 
-An example where the `CLUSTERNAME` from the prior `helm install` or `helm upgrade` was `k8ssandra`:
+To use Prometheus for monitoring, you need to have the Prometheus-operator installed on your Kubernetes (k8s) cluster.
+The Prometheus-operator installs the ServiceMonitor CRD, which is the integration point we use to tell Prometheus how to find the Stargate and Cassandra pods and what endpoints on those pods to scrape.
 
-```bash
-kubectl rollout status statefulset k8ssandra-dc1-default-sts
-```
+### Install Prometheus-operator
 
-**Output:**
+*Skip to the next section if you already have the Prometheus-operator installed*  
+  
+Install the Prometheus operator by running the following command:
 
-```bash
-Waiting for 1 pods to be ready...
-Waiting for 1 pods to be ready...
-partitioned roll out complete: 1 new pods have been updated...
 ```
-
-If you submit the command above too quickly after the `helm install` or `helm upgrade` you may get an error that no statefulset with that name exists; in which case, you can run the `kubectl rollout status` command again.  
-
-You should also check the `stargate` pod status before proceeding. Example:
-
-```bash
-kubectl rollout status deployment k8ssandra-dc1-stargate
+kubectl create -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/master/bundle.yaml
 ```
 
-**Output:**
+This will install all the CRDs along with a `prometheus` service account (SA), a cluster role and a cluster role binding to the `prometheus` SA.
```

Comment:
I'm not a fan of installing into the `default` namespace.

Reply:
> Should that be
> 
> to the default namespace.

I don't think so 🤔 The cluster role is bound to the SA, not the namespace. Should I say "binding for" instead of "binding to" ?

> I'm not a fan of installing into the default namespace.

me neither, but the prometheus-operator is not making this easy. I don't want to overcomplicate the documentation with details on how to do this.

Reply:
kube-prometheus-stack installs the prom operator into the `monitoring` ns if that helps.